packages/tradershub/src/features/cfd/screens/TradeScreen/TradeDetailsItem/TradeDetailsItem.tsx

```diff
@@ -22,13 +23,17 @@ const TradeDetailsItem = ({ className, label, value, variant = 'clipboard' }: TT
                 className
             )}
         >
-            <Text colorStyle='subtle' size='sm'>
-                {label}
-            </Text>
+            {label && <Text size='sm'>{label}</Text>}
             <div className='flex items-center gap-x-400'>
-                <Text bold={variant !== 'info'} size='sm'>
-                    {value}
-                </Text>
+                {variant === 'info' ? (
+                    <Text color='less-prominent' size='sm'>
+                        {value}
+                    </Text>
+                ) : (
+                    <Text size='sm' weight='bold'>
+                        {value}
+                    </Text>
+                )}
```

Comment:
Rather than using props can we just use inline classNames instead like
```suggestion
                <Text size='sm' className={qtMerge('font-bold', { 'text-less-prominent font-normal': variant === 'info' })}>
                      {value}
                </Text>
```

Reply:
@aizad-deriv in my opinion, using ternary operator here is better and easier to understand